src/pages/term/Term.module.sass

```
@@ -0,0 +1,6 @@
+.button-list
+    display: flex
+    justify-content: space-between
```

- It's not a list, it's a group.
- I think the flex content should be aligned to the right, so the "next" button won't be so far from the main UX area.
```suggestion
.button-list
    display: flex
    gap: 2rem
```